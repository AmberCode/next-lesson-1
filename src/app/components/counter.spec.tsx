import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import Counter from '../components/counter';
 
describe('Counter', () => {
  it('should render', () => {
    const { container } = render(<Counter />);
    expect(container).toMatchSnapshot();
  });

  it('should have btn', () => {
    render(<Counter />)
    const btn = screen.getByTestId('buttonCount');
    expect(btn).toBeInTheDocument();
  });
});
