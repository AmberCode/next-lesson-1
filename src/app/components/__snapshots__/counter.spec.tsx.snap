// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter should render 1`] = `
<div>
  <div>
    <p>
      You clicked 
      0
       times
    </p>
    <button
      data-testid="buttonCount"
    >
      Click Me
    </button>
  </div>
</div>
`;
